xml/security_polkit.xml

```diff
@@ -401,30 +269,29 @@
      </term>
      <listitem>
       <para>
-       Designed for machines administered centrally<phrase os="sles">. It
-       is active by default.</phrase>
+       More conservative authorization settings that reduce possible attack surface
+       at the expense of user experience in some areas.
       </para>
      </listitem>
     </varlistentry>
    </variablelist>
    <para>
-    To switch between the two sets of default privileges, adjust the value
-    of <varname>POLKIT_DEFAULT_PRIVS</varname> to either
-    <literal>restrictive</literal> or <literal>standard</literal> in
-    <filename>/etc/sysconfig/security</filename>. Then run the command
-    <command>set_polkit_default_privs</command> as &rootuser;.
+    To switch the active polkit profile, edit <filename>/etc/sysconfig/security</filename>
+    and adjust the value of <varname>POLKIT_DEFAULT_PRIVS</varname> to one of
+    <literal>easy</literal>, <literal>standard</literal> or <literal>restrictive</literal>.
+    Then run the command <command>set_polkit_default_privs</command> as &rootuser;.
    </para>
    <para>
-    Do not modify the two files in the list above. To define your
-    own custom set of privileges, use
+    Do not modify the profile settings in the files listed above. To define your
+    own custom &pk; settings, use
```

Comment:
that feels a bit misplaced. Move it up to where the directories are described?

Reply:
This doesn't really fit in the earlier section. The earlier section is about the polkit policy files and therefore Polkit standard configuration files. While this part here is about our polkit-default-privs. I don't think it is really misplaced, since this is the section where SUSE default privileges are introduced the first time.